backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceTypeSpecimenTypeMapping.java

```diff
@@ -0,0 +1,28 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "device_specimen_type")
```

Comment:
We currently have a `deviceSpecimenType` entity, this is a replacement. Notice how this has UUID's instead of the entity classes, which prevents the extra joins and also doesnt have the extra id,created_by,created_id,updated_at,updated_by which the main ticket is removing them since they shouldnt exist on a join table